### PR TITLE
Update paths to be universal

### PR DESCRIPTION
**Pull request by mdeguzis**
Other alternative OS's, namely ChimeraOS can and do use Deck Loader. Update the paths to reflect the user home directory.

### CODE REVIEW

**Review by GedasFX — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -104,9 +104,9 @@ On the plugin sidebar there is a new option to change behavior of plugin to not
 Before your first sync, first you will need to manually run the following command in console:
 
 ```
-/home/deck/homebrew/plugins/decky-cloud-save/rclone bisync --filter-from /home/deck/homebrew/settings/decky-cloud-save/sync_paths_filter.txt / backend:decky-cloud-save --copy-links --resync
+~/homebrew/plugins/decky-cloud-save/rclone bisync --filter-from ~/homebrew/settings/decky-cloud-save/sync_paths_filter.txt / backend:decky-cloud-save --copy-links --resync
```
[the remark this answers is not in the document]
Comment (GedasFX) [reply README.md:107]: I'm sorry I didn't know how github reviews worked... I left a comment when you made the PR here and didnt realize it was pending. Thought I got ghosted 😅

[the remark this answers is not in the document]
Comment (mdeguzis) [reply README.md:107]: This isn't a relative path change. This just ensures a dynamic home is used before the rest of the path.

[the remark this answers is not in the document]
Comment (GedasFX) [reply README.md:107]: Yeah, looking into this now, this looks alright

**Review by GedasFX — APPROVED** [review]

**Comment by Emiliopg91** [comment]
@GedasFX why not using decky_plugin.pyi consts?

**Comment by GedasFX** [comment]
@Emiliopg91 this is in readme.

**Pull request merged**